server/sonar-server/src/test/resources/org/sonar/server/component/ws/SearchViewComponentsActionTest/shared.xml

```diff
@@ -4,11 +4,13 @@
   <group_roles id="1"
                group_id="[null]"
                resource_id="100"
-               role="user"/>
+               role="user"
+               organization_uuid="[null]"/>
```

Comment:
why add `organization_uuid="[null]"` to all these DBUnit file used solely for initialization when, without modification, the behaviour would be the same?